### PR TITLE
Implement the top score panel contents

### PR DESCRIPTION
**Pull request by smoogipoo**
Prereqs:
- [x] https://github.com/ppy/osu/pull/8308

image

### CODE REVIEW

File: osu.Game/Screens/Ranking/Expanded/ExpandedPanelTopContent.cs [hunk 1]
```diff
@@ -1,15 +1,64 @@
...
+                Direction = FillDirection.Vertical,
+                Children = new Drawable[]
+                {
+                    new UpdateableAvatar(user)
+                    {
+                        Anchor = Anchor.TopCentre,
+                        Origin = Anchor.TopCentre,
+                        Size = new Vector2(80),
+                        CornerRadius = 20,
+                        CornerExponent = 2.5f,
```
Comment (smoogipoo) [inline osu.Game/Screens/Ranking/Expanded/ExpandedPanelTopContent.cs]: Just remembered I did this, not sure if you want it in like this.

Comment (peppy) [reply osu.Game/Screens/Ranking/Expanded/ExpandedPanelTopContent.cs]: which part? the corner exponent?

Comment (smoogipoo) [reply osu.Game/Screens/Ranking/Expanded/ExpandedPanelTopContent.cs]: Corner exponent, yes

Comment (peppy) [reply osu.Game/Screens/Ranking/Expanded/ExpandedPanelTopContent.cs]: seems sane enough

**Pull request merged**